src/Skia/Avalonia.Skia/Helpers/DrawingContextHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Avalonia.Platform;
+using Avalonia.Rendering;
+using SkiaSharp;
+
+namespace Avalonia.Skia.Helpers
+{
+    public class DrawingContextHelper
+    {
+        /// <summary>
+        /// Wrap Skia canvas in drawing context so we can use Avalonia api to render to external skia canvas
+        /// this is useful in scenarios where canvas is not controlled by application, but received from another non avalonia api
+        /// like: SKCanvas canvas = SKDocument.BeginPage(...);
+        /// </summary>
+        /// <param name="canvas"></param>
+        /// <param name="dpi"></param>
+        /// <param name="visualBrushRenderer"></param>
+        /// <returns>DrawingContext</returns>
+        public static IDrawingContextImpl WrapSkiaCanvas(SKCanvas canvas, Vector dpi, IVisualBrushRenderer visualBrushRenderer = null)
```

Comment:
Should we return the `Impl` here? It's also technically a private API.

Reply:
I'll be more happy to return the impl, here, as it's a public interface and suits for my use case. 
But if you strongly believe we shouldn't return the impl, i can change it to return `DrawingContext` and will use some workarounds to get the impl, as in my scenario i need and use the impl.

Reply:
This is some kind of "I know what I am doing API" and `DrawingContext` can be constructed from a `IDrawingContextImpl` so this should be fine.